Makefile

```diff
@@ -65,30 +70,32 @@ dist: clean browserpass-linux64 browserpass-darwinx64 browserpass-openbsd64 brow
 #######################
 # For user installation
 
+.PHONY: configure
+configure:
+	sed -i "s|%%replace%%|$(BIN_DIR)/$(BIN)|" browser-files/chromium-host.json
+	sed -i "s|%%replace%%|$(BIN_DIR)/$(BIN)|" browser-files/firefox-host.json
```

Comment:
FYI @Infinisil I realized that there was one more place where `/usr/bin` was configured - when setting the path to the binary in hosts files. I introduced a separate goal for this, because on Arch Linux I need to use different `DESTDIR` in `configure` and `install` targets.

So installation now is `make configure` followed by `make install`.